### PR TITLE
Polyfill for String.prototype.normalize()

### PR DESCRIPTION
**Pull request by AdrienClairembault**
Compatibility fix for IE (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize#Browser_compatibility).

Polyfill used : https://github.com/walling/unorm

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Internal ref | [16088](https://support.teclib.com/front/ticket.form.php?id=16088)

### CODE REVIEW

File: lib/unorm/unorm.js [hunk 1]
```diff
@@ -0,0 +1,894 @@
+(function (root) {
+   "use strict";
+
+/***** unorm.js *****/
+
+/*
+ * UnicodeNormalizer 1.0.0
+ * Copyright (c) 2008 Matsuza
+ * Dual licensed under the MIT (MIT-LICENSE.txt) and GPL (GPL-LICENSE.txt) licenses.
+ * $Date: 2008-06-05 16:44:17 +0200 (Thu, 05 Jun 2008) $
```
Comment (trasher) [inline lib/unorm/unorm.js]: This one is quite old. Isn't there any more recent lib?

I'm not fan of loading this script everywhere for a specific IE version issue... But on another hand, I'm not sure we can get browser and version on a secure way...

File: js/common.js [hunk 2]
```diff
@@ -30,6 +30,14 @@
  */
 var timeoutglobalvar;
 
+if (typeof(String.prototype.normalize) !== 'function') {
+   $.ajax({
+      type: "GET",
+      url: "../lib/unorm/unorm.js",
```
Comment (cedric-anne) [inline js/common.js]:
```suggestion
      url: CFG_GLPI.root_doc+ "/lib/unorm/unorm.js",
```

File: lib/unorm/unorm.js (hunk 1 above)
Comment (orthagh) [reply lib/unorm/unorm.js]: Header not updated upstream, this is the last version published yesterday.
Also, your concern about ie only should be now addressed

Comment (trasher) [reply lib/unorm/unorm.js]: Okay thanks for explanations :)

**Comment by cedric-anne** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/ :D

**Pull request merged**